docs/src/explanations/Documentation-of-functions.md

```diff
@@ -0,0 +1,341 @@
+---
+
+layout: docs
+
+title:  "Documention of functions"
+
+section: "chisel3"
+
+---
+
+# Documentation of Functions := vs <>
+
+The difference in the function of := and <> has been a major source of debate in lots of conversations
+
+This document explains the difference by setting up an experiment using Scastie examples + DecoupledIO examples to put the functions to the test.
+
+
+
+### Experiment
+
+```scala mdoc
+// Imports used by the following examples
+import chisel3._
+import chisel3.util.DecoupledIO
+```
+
+The diagram for the experiment can be viewed **[here](https://docs.google.com/document/d/14C918Hdahk2xOGSJJBT-ZVqAx99_hg3JQIq-vaaifQU/edit?usp=sharing)**
```

Comment:
we should figure out how to embed the actual diagram... we'll need to read the markdown docs

Reply:
I will go through that and see how I can put it in